mino/option.go

```diff
@@ -95,3 +109,17 @@ func ListFilter(indices []int) FilterUpdater {
 		filters.Indices = indices
 	}
 }
+
+// RandomFilter chooses 'count' random elements.
+func RandomFilter(count int) FilterUpdater {
+	return func(filters *Filter) {
+		if len(filters.Indices) < count {
+			return
```

Comment:
doesn't that mean that

1. the caller cannot know that the elements have not been chosen at random if |filter.Indices| < count
2. the caller cannot distinguish between randomly chosen elements and the re-ordered initial array if |filter.Indice| == count

unless the caller explicitly tests for these two cases? is that a problem?

Reply:
The goal of the `RandomFilter` is to choose `count` random elements of the `Indices`. It returns the `Indices` sorted.

Given these constraints, if the length of the `Indices` is `<=` count, there is nothing to be done.

So I think I only need to update the `<=` and the comment...

Does that look reasonable?